Gemfile

```diff
@@ -41,12 +41,14 @@ gem "fast_jsonapi", "~> 1.5" # Serializer for Ruby objects
 gem "fastly", "~> 1.15" # Client library for the Fastly acceleration system
 gem "fastly-rails", "~> 0.8" # Fastly dynamic caching integration for Rails
 gem "feedjira", "~> 3.1" # A feed fetching and parsing library
+gem "field_test", "~> 0.3" # A/B testing
 gem "figaro", "~> 1.1" # Simple, Heroku-friendly Rails app configuration using ENV and a single YAML file
 gem "fog-aws", "~> 3.5" # 'fog' gem to support Amazon Web Services
 gem "front_matter_parser", "~> 0.2" # Parse a front matter from syntactically correct strings or files
 gem "gemoji", "~> 4.0.0.rc2" # Character information and metadata for standard and custom emoji
 gem "gibbon", "~> 3.3" # API wrapper for MailChimp's API
 gem "google-api-client", "~> 0.37" # Client for accessing Google APIs
+gem "groupdate", "~> 5.0" # Group collections by date
```

Comment:
I have been trying to avoid using this gem unless necessary as Rails already has native group by with this syntax:

`group("DATE(created_at)")`

See:

```ruby
[12] pry(main)> Article.group("DATE(created_at)").count
   (2.4ms)  SELECT COUNT(*) AS count_all, DATE(created_at) AS date_created_at FROM "articles" GROUP BY DATE(created_at)
=> {Mon, 24 Feb 2020=>25, Tue, 25 Feb 2020=>1}
```

This is what we use in `AnalyticsService`

Reply:
Removed ✅

Reply:
> it's a little weird that we have to deploy a change to close an experiment

I agree. I think we could move this to an environment variable or otherwise redefine how this is done, but I think starting with the default for how the gem likes to do it makes sense.

Reply:
One good thing about having to deploy to close an experiment is you have a natural point at which to remove the experiment/fieldtest/a-b test code for the experiment.